storage/database/how-to/connect-database-private-network.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: How to connect a Database Instance to a Private Network
+description: This page explains how to connect a Database Instance to a Private Network
+dates:
+  validation: 2021-11-15
+  posted: 2019-11-15
+---
+
+[Private Networks](/network/vpc/quickstart/) allows you to configure your own Layer-2 network with IPv4 adresses. Thanks to this feature, you enhance the security of your systems architecture by isolating it from the Internet. 
+
+This feature comes with a significant change in the architecture of Scaleway's Managed Databases: when using Private Networks, your application connects directly to your Database nodes without going through a Load Balancer. 
+This improves performance by reducing the latency between your nodes. It also increases the security of your databases, as Instances in your Private Network can directly to your Database Instance, without passing through the public Internet.
```

Comment:
I think there's a word missing after "as Instances in your Private Network can directly"